health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AddOrEditMapping.js

```
@@ -0,0 +1,177 @@
+    useImperativeHandle(ref, () => ({
+        getData: () => {
+        return newdata;
+        },
+    }));
+
+    const renderInput = (column) => {
+        if (column?.type === "number") {
+            return (
+                <div key={column.name} style={{ marginBottom: "1rem" }}>
+                    <div style={{display:"flex"}}>
+                        <label>{column.description}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)